src/styles/LandingPage.module.less

```diff
@@ -37,17 +38,44 @@
     right: 0;
     left: 0;
     width: 75.3vw;
-    color: #373737;
+    color: #ffffff;
     font-size: 3.644rem; 
-    font-weight: 900;
+    font-weight: 400;
     text-align: left;
-    line-height: 7.822rem;
-    padding-left: 5%;
+    line-height: 7.157rem;
+    letter-spacing: 0.1rem;
+    padding-left: 0%;
 }
 
 // Use strong to give first word emphasis 
 .LandingPageOverlayText strong{ 
-    color: #0b688a;
+    color: #ffffff;
+    font-size: 4.063rem; 
+    font-weight: bold;
+}
+
+.LandingPageOverlayText button{ 
+    background-color: #3b9ec2;
+    line-height: 2.15;
+    text-align: center;
+    font-size: 1.7rem; 
+    font-weight: bold;
+    color: #fff;
+    border: 0;
+    border-radius: 3rem;
+    width: 26rem;
+    height: 4.79542334076rem;
+    padding-top: 0.5rem;
+    transition-duration: 0.4s;
+}
+
+.LandingPageOverlayText button:hover{ 
+    background-color: #3fcafc; 
+}
+
+.ButtonWrapper{
+    padding-top: 5rem;
```

Comment:
I think you might need to add padding to the bottom as well? Maybe not in this class but somewhere else. Right now that text of the button is not centered. Inspecting the button it just appears that the text has less padding on the bottom then the top.

Reply:
I think that bit of code was from something else, it doesn't actually do anything and I just deleted it. For some reason the text isn't centering inside the button.